draft-ietf-moq-warp.md

```diff
@@ -1171,6 +1172,117 @@ This example shows drone GPS coordinates synched with the start of each Group.
 
 # Workflow
 
+## URL construction & interpretation
+A WARP URL is a String with the following components
+
+moqt:// + authority + path + query
+
+* The URL MUST conform with {{RFC3986}}.
+* The protocol is moqt and is required.
+* The authority is required and holds the host and optional port. The default port
+  will be 443. The authority holds the information needed for the client to connect
+  to the distribution service, using either WebTransport or Raw QUIC.
+* The path is optional. If present, it contains information that might be used by
+  the server in initializing and configuring the connection.
+* The query is optional. If present, it holds key-value data. Certain keys are reserved
+  by this specificaiton to hold special data. Multiple key/value pairs MAY be included,
+  separated by an ampersand. Query options MUST be stripped from the path when the path
+  is sent to the server at conneciton establishment. Query arguments are intended only
+  for the player and MUST NOT be transmitted to the server.
+
+An example URL is shown below:
+
+moqt://example.com/relay-app/relayID?c=customerID/broadcastID/catalog&c4m=12345
+
+
+### Reserved query arguments
+
+Table 5 defines reserved key names for the query portion of the URL. Keynames are
+case-sensitive.
+
+| Name            |                Description                       |
+|:================|:=================================================|
+| ns              | The Namespace of the track                       |
+| t               | The Name of a non-catalog track                  |
+| wallclock-range | A subclip defined by a wallclock time range      |
+| mediatime-range | A subclip defined by a media time range          |
+| location-range  | A subclip defined by a MOQT Location range       |
+| c4m             | A base64 encoded C4M token                       |
+
+ * ns - the Namespace of the track. The '/' character in the String defines
+   tuple field boundaries and is not included in the tuple fields. The '/' character
+   MUST NOT be used other than to define tuple boundaries. A closing '/' for the
+   last tuple field MUST NOT be included. This key MUST be included if 't' is present.
+ * t - the track name of a non-catalog track. This field MUST NOT be included if the
+   URL is referencing a catalog track.
+ * wallclock-range - a range defined by start and end wallclock times, each expressed
+   as milliseconds since Unix Epoch and separated by a "-" dash.
+ * mediatime-range - a range defined by start and end media times, each expressed
+    as milliseconds and separated by a "-" dash.
```

Comment:
I think wording from [7.1. ][Media Timeline track payload](https://moq-wg.github.io/warp-streaming-format/draft-ietf-moq-warp.html#name-media-timeline-track-payloa) may be taken here.

"media presentation timestamp, rounded to the nearest millisecond"

Sidenote, why are we not using timescale, if catalog defines it? Now I am confusing myself. Does even LOC have PTS?